config/.editorconfig

```diff
@@ -0,0 +1,32 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+
+[*.hbs]
+insert_final_newline = false
+
+[*.{js,ts,jsx,yml}]
```

Comment:
Do we want json to be indented with 2 spaces? I didn't add it because I thought we wanted it with 4 spaces (except for the `package.json`)